### DISCUSSION TITLE
Schema directive

### DISCUSSION THREAD

**Discussion by JohnnyMaxK** [dotansimha/graphql-code-generator#5649] [Q&A] [answered]
When I test a schema directive on a field or a type I saw that the introspection generated from the schema does not take it the schema directives.
I need to manage in my code generation from the schema the schema directives to be able to generate the attributes on my properties or classes.
In my specific case, it is to manage authorizations.
Can you tell if it’s a bug, please?

**Comment by ardatan** [comment, marked as answer]
Currently it is not possible to have directives in introspection (this is how GraphQL Introspection work). This is not related to GraphQL Codegen. You can only have directives as an SDL(using schema-ast).
